### DISCUSSION TITLE
Data sharing id

### DISCUSSION THREAD

**Discussion by yaoshuyu** [milvus-io/milvus#4779] [Q&A and General discussion]
If I use milvus for face matching, there are two face data with and without a mask under the same id. After .search (top=1), will it automatically match the closer result or the earlier result?
such as 
id=1012 | […………] ←earlier,without mask
id=1012 | […………] ←with mask

**Comment by shengjun1985** [comment]
At present, we don't have ID duplicate checking.
As long as it is not deleted, we will return the closer item.
